<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="gamePage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="Screen" name="screenWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Current disk:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="currentDiskLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>---</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="disassemblerPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="disassemblerBrowser">
          <property name="font">
           <font>
            <family>Courier 10 Pitch</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Line no.:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineNoField">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="textPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Eamon</string>
    </property>
    <widget class="QMenu" name="menuCharacter_Utilities">
     <property name="title">
      <string>Character Utilities</string>
     </property>
     <addaction name="actionDisplay_Characters"/>
     <addaction name="actionView_Remove_Character"/>
     <addaction name="actionResurrect_Character"/>
     <addaction name="actionCharacter_File_Maintenance"/>
    </widget>
    <addaction name="actionRun_Disk"/>
    <addaction name="actionRun"/>
    <addaction name="actionInsert_Game_Disk"/>
    <addaction name="separator"/>
    <addaction name="menuCharacter_Utilities"/>
    <addaction name="actionRun_Utility"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionEditor"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Characters"/>
    <addaction name="actionImport_Characters"/>
    <addaction name="separator"/>
    <addaction name="actionExtract_Disk"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_Player_s_Manual"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun_Disk"/>
   <addaction name="actionRun"/>
   <addaction name="actionInsert_Game_Disk"/>
   <addaction name="separator"/>
   <addaction name="actionCharacter_File_Maintenance"/>
   <addaction name="actionShow_Player_s_Manual"/>
   <addaction name="separator"/>
   <addaction name="actionEditor"/>
   <addaction name="actionScreenshot"/>
   <addaction name="actionPreferences"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/configure.png</normaloff>:/resources/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset Main Hall...</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/go-home.png</normaloff>:/resources/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Enter Main Hall</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShow_Player_s_Manual">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/page-3sides.png</normaloff>:/resources/icons/page-3sides.png</iconset>
   </property>
   <property name="text">
    <string>Show Player's Manual</string>
   </property>
  </action>
  <action name="actionDisplay_Characters">
   <property name="text">
    <string>Display Characters</string>
   </property>
  </action>
  <action name="actionExtract_Disk">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/archive-extract.png</normaloff>:/resources/icons/archive-extract.png</iconset>
   </property>
   <property name="text">
    <string>Extract Disk...</string>
   </property>
  </action>
  <action name="actionCharacter_File_Maintenance">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/user-group-properties.png</normaloff>:/resources/icons/user-group-properties.png</iconset>
   </property>
   <property name="text">
    <string>Character File Maintenance</string>
   </property>
  </action>
  <action name="actionResurrect_Character">
   <property name="text">
    <string>Resurrect Character</string>
   </property>
  </action>
  <action name="actionInsert_Game_Disk">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/drive-optical.png</normaloff>:/resources/icons/drive-optical.png</iconset>
   </property>
   <property name="text">
    <string>Insert Game Disk...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRun_Utility">
   <property name="text">
    <string>Run Utility...</string>
   </property>
  </action>
  <action name="actionView_Remove_Character">
   <property name="text">
    <string>View / Remove Character</string>
   </property>
  </action>
  <action name="actionExport_Characters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Characters...</string>
   </property>
  </action>
  <action name="actionImport_Characters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Characters...</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/camera-photo.png</normaloff>:/resources/icons/camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>Screenshot...</string>
   </property>
  </action>
  <action name="actionRun_Disk">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/run.png</normaloff>:/resources/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Game...</string>
   </property>
   <property name="toolTip">
    <string>Directly run a game disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="icon">
    <iconset resource="eamon.qrc">
     <normaloff>:/resources/icons/accessories-text-editor.png</normaloff>:/resources/icons/accessories-text-editor.png</iconset>
   </property>
   <property name="text">
    <string>Editor...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QFrame</extends>
   <header>screen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="eamon.qrc"/>
 </resources>
 <connections/>
</ui>
